.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+
+language: go
+
+go:
+- 1.5.4
```

Comment:
People might still build it with 1.5, we do the same with other components.
Moreover, travis builds are fast, so it doesn't cost much time.